src/WinRT.Runtime/ExceptionHelpers.cs

```diff
@@ -98,6 +98,29 @@ private static bool Initialize()
             return true;
         }
 
+        /// <summary>
+        /// Unhandled WinRT server exception event.
+        /// </summary>
+        public static event EventHandler<UnhandledWinRTServerExceptionEventArgs> UnhandledWinRTServerException;
+
+        public class UnhandledWinRTServerExceptionEventArgs : EventArgs
+        {
+            public Exception Exception { get; }
+            public bool Handled { get; set; }
+
+            public UnhandledWinRTServerExceptionEventArgs(Exception exception)
+            {
+                Exception = exception;
+            }
+        }
+
+        public static bool TryHandleWinRTServerException(object sender, Exception ex)
+        {
+            UnhandledWinRTServerExceptionEventArgs args = new UnhandledWinRTServerExceptionEventArgs(ex);

```

Comment:
Definitely don't do this, this will always allocate the args even if there's no handler.

Reply:
Resolved.